src/libs/settings.js

```diff
@@ -9,24 +9,49 @@ export async function getSettings() {
 		//  Enable the logging automatically if we cannot determine the user preference.
 		settings = {};
 	}
+	return settings;
 }
 
 export function isLogging() {
 	return settings.logging;
 }
 
 export function statusVisibility() {
-	return settings.statusVisibility;
+	return settings.proplate.statusVisibility;
```

Comment:
would it be useful (to introduce) a version marker of some sort for settings in general?
not for new settings key which can use the default value but for such structural changes to allow migration.
maybe versioning the settings is too much for KISS reasons for now.
in this case we can detect and migrate without it, or let migration out and so users would loose their settings this time. (shouldn't happen too often)

Reply:
while i think it may be a little annoying for users to have to re-set their settings from time to time we don't push updates that change the structure of things that often. and also i'm lazy '^^

I'll gladly review a PR for it, but admittedly oftentimes i just wanna have the shiny new feature :3c